packages/terra-card/src/Card.scss

```diff
@@ -0,0 +1,27 @@
+
+/* stylelint-disable selector-class-pattern  */
+.card {
```

Comment:
You should probably discuss with @neilpfeiffer to determine if any of these values should be themeable.